.travis.yml

```diff
@@ -43,15 +43,32 @@ matrix:
       env: BUILD="base" TEST_TARGET="scalding-jdbc scalding-json"
       script: "scripts/run_test.sh"
 
-    - scala: 2.11.4
+    - scala: 2.11.5
       env: BUILD="base" TEST_TARGET="scalding-jdbc scalding-json"
       script: "scripts/run_test.sh"
 
+# not committed yet
```

Comment:
I think I uncommented in the follow up (I think). I'm just as worried we won't remember to add it (since that happens to). I'll do what you prefer.